SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  ViewController.swift
+//  SupportClient
+//
+//  Created by Omar Rasheed on 2/15/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    
+    var newConversationButton: UIButton!
+    var feedbackLabel: UILabel!
+    var searchController: UISearchController! // TODO: For later
+    var feedbackTableView: UITableView!
```

Comment:
lets change all of these to `private let`, i.e.: `private let newConversationButton = UIButton()`
the reason is so that we make sure that other classes can access these variables and we don't end up changing them anywhere else in the code

Reply:
Im gonna pull a @young-k but lets alphabetize too lmfao